### PR TITLE
Adding inputs metadata calculation (#45).

### PR DESCRIPTION
**Pull request by ola-rozenfeld**

### CODE REVIEW

File: go/pkg/rexec/rexec_test.go [hunk 1]
```diff
@@ -36,8 +36,10 @@ func TestExecCacheHit(t *testing.T) {
 	res, meta := e.Client.Run(context.Background(), cmd, opt, oe)
 
 	wantMeta := &command.Metadata{
-		CommandDigest: cmdDg,
-		ActionDigest:  acDg,
+		CommandDigest:    cmdDg,
```
Comment (ramymedhat) [inline go/pkg/rexec/rexec_test.go]: Can we also have a test where the input file bytes are not zero?

Comment (ola-rozenfeld) [reply go/pkg/rexec/rexec_test.go]: Sure, added an input file to one of the tests.
